src/routes.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Route, Switch } from "react-router-dom";
+
+import LoginFormContainer from "./containers/LoginFormContainer";
+import EditProfileContainer from "./containers/EditProfileContainer";
+import HomePage from "./components/HomePage/HomePage";
+import Navbar from "./components/Navbar/Navbar";
+
+const routes = (
+  <div className="App">
+    <Navbar />
+    <div className="content">
+      <Switch>
+        <Route exact path="/" component={HomePage} />
+        <Route path="/login" component={LoginFormContainer} />
+        <Route path="/edit-profile" component={EditProfileContainer} />
```

Comment:
could you use const for paths?

Reply:
They're like this since #7 tho 🤔 
I can ofc change it as you suggestted, but I'd rather make another refactor / cleanup PR after finishing current sprint